src/redux/reducers/initialState.js

```diff
@@ -80,7 +80,9 @@ const initialState = {
     liked: false,
     disliked: false,
     error: ""
-  }
+  },
+  profile: null,
```

Comment:
`null`? what does that mean?

Reply:
@kimotho-njoki I will initialize it to an empty object.